### PR TITLE
chore(docs): update BridgeCrew links in metadata to our local docs link 

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Update BridgeCrew links in metadata to our local docs link.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/developer-guide/checks.md [hunk 1]
```diff
@@ -249,11 +249,11 @@ Each Prowler check has metadata associated which is stored at the same level of
     # Code holds different methods to remediate the FAIL finding
     "Code": {
       # CLI holds the command in the provider native CLI to remediate it
-      "CLI": "https://docs.bridgecrew.io/docs/public_8#cli-command",
+      "CLI": "https://docs.prowler.com/checks/public_8#cli-command",
```
Comment (puchy22) [inline docs/developer-guide/checks.md:252]: I think the link you are referring in this file is https://docs.prowler.com/checks/aws/public-policies/public_8/

File: tests/lib/check/check_loader_test.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def get_custom_check_metadata(self):
             Remediation=Remediation(
                 Code=Code(
                     NativeIaC="",
-                    Terraform="https://docs.bridgecrew.io/docs/bc_aws_s3_20#terraform",
+                    Terraform="https://docs.prowler.com/checks/aws/general-policies/bc_aws_s3_20#terraform",
```
Comment (puchy22) [inline tests/lib/check/check_loader_test.py]: This link is not correct, please check it

File: tests/lib/check/custom_checks_metadata_test.py [hunk 3]
```diff
@@ -46,7 +46,7 @@ def get_custom_check_metadata(self):
             Remediation=Remediation(
                 Code=Code(
                     NativeIaC="",
-                    Terraform="https://docs.bridgecrew.io/docs/bc_aws_s3_20#terraform",
+                    Terraform="https://docs.prowler.com/checks/aws/general-policies/bc_aws_s3_20#terraform",
```
Comment (puchy22) [inline tests/lib/check/custom_checks_metadata_test.py]: This link is not correct, please check it

File: tests/lib/check/fixtures/bulk_checks_metadata.py [hunk 4]
```diff
@@ -23,7 +23,7 @@
             Code=Code(
                 NativeIaC="",
                 Terraform="",
-                CLI="https://docs.bridgecrew.io/docs/networking_5#cli-command",
+                CLI="https://docs.prowler.com/checks/networking_5#cli-command",
```
Comment (puchy22) [inline tests/lib/check/fixtures/bulk_checks_metadata.py:26]: This file links isn't working, please check it

**Review by puchy22 — CHANGES_REQUESTED** [review]

**Comment by puchy22** [comment]
Hi @sergargar, I was checking all the links on metadata were well written. This are my findings of not 200 status code please check it:
```
Provider: Azure
Check: postgresql_flexible_server_allow_access_services_disabled.metadata.json
https://docs.prowler.com/checks/ensure-allow-access-to-azure-services-for-postgresql-database-server-is-disabled#terraform -- 404
----------------------
Provider: Azure
Check: monitor_storage_account_with_activity_logs_cmk_encrypted.metadata.json
https://docs.prowler.com/checks/ensure-that-storage-accounts-use-customer-managed-key-for-encryption#terraform -- 404
----------------------
Provider: Azure
Check: monitor_storage_account_with_activity_logs_is_private.metadata.json
https://docs.prowler.com/checks/ensure-the-storage-container-storing-the-activity-logs-is-not-publicly-accessible#terraform -- 404
----------------------
Provider: Azure
Check: app_ensure_python_version_is_latest.metadata.json
https://docs.prowler.com/checks/ensure-that-python-version-is-the-latest-if-used-to-run-the-web-app -- 404
----------------------
Provider: Azure
Check: app_ensure_java_version_is_latest.metadata.json
https://docs.prowler.com/checks/ensure-that-java-version-is-the-latest-if-used-to-run-the-web-app#terraform -- 404
----------------------
Provider: Azure
Check: app_ftp_deployment_disabled.metadata.json
https://docs.prowler.com/checks/ensure-ftp-deployments-are-disabled#terraform -- 404
----------------------
Provider: Azure
Check: app_http_logs_enabled.metadata.json
https://docs.prowler.com/checks/ensure-that-app-service-enables-http-logging#terraform -- 404
----------------------
Provider: Azure
Check: app_ensure_php_version_is_latest.metadata.json
https://docs.prowler.com/checks/ensure-that-php-version-is-the-latest-if-used-to-run-the-web-app#terraform -- 404
----------------------
Provider: Azure
Check: keyvault_recoverable.metadata.json
https://docs.prowler.com/checks/ensure-the-key-vault-is-recoverable#terraform -- 404
----------------------
Provider: Azure
Check: network_udp_internet_access_restricted.metadata.json
https://docs.prowler.com/checks/ensure-that-udp-services-are-restricted-from-the-internet#terraform -- 404
----------------------
Provider: Azure
Check: storage_secure_transfer_required_is_enabled.metadata.json
https://docs.prowler.com/checks/ensure-that-storage-account-enables-secure-transfer -- 404
----------------------
Provider: Azure
Check: storage_ensure_encryption_with_customer_managed_keys.metadata.json
https://docs.prowler.com/checks/ensure-that-storage-accounts-use-customer-managed-key-for-encryption#terraform -- 404
----------------------
Provider: Azure
Check: storage_blob_public_access_level_is_disabled.metadata.json
https://docs.prowler.com/checks/ensure-that-storage-accounts-disallow-public-access#terraform -- 404
----------------------
Provider: Azure
Check: vm_ensure_using_managed_disks.metadata.json
https://docs.prowler.com/checks/ensure-virtual-machines-are-utilizing-managed-disks#terraform -- 404
----------------------
Provider: Azure
Check: mysql_flexible_server_minimum_tls_version_12.metadata.json
https://docs.prowler.com/checks/ensure-mysql-is-using-the-latest-version-of-tls-encryption#terraform -- 404
----------------------
Provider: Azure
Check: sqlserver_va_periodic_recurring_scans_enabled.metadata.json
https://docs.prowler.com/checks/ensure-that-va-setting-periodic-recurring-scans-is-enabled-on-a-sql-server#terraform -- 404
----------------------
Provider: Azure
Check: sqlserver_va_emails_notifications_admins_enabled.metadata.json
https://docs.prowler.com/checks/ensure-that-va-setting-also-send-email-notifications-to-admins-and-subscription-owners-is-set-for-an-sql-server#terraform -- 404
----------------------
Provider: Azure
Check: sqlserver_va_scan_reports_configured.metadata.json
https://docs.prowler.com/checks/ensure-that-va-setting-send-scan-reports-to-is-configured-for-a-sql-server#terraform -- 404
----------------------
Provider: Azure
Check: sqlserver_azuread_administrator_enabled.metadata.json
https://docs.prowler.com/checks/ensure-that-azure-active-directory-admin-is-configured#terraform -- 404
----------------------
Provider: Azure
Check: sqlserver_vulnerability_assessment_enabled.metadata.json
https://docs.prowler.com/checks/ensure-that-vulnerability-assessment-va-is-enabled-on-a-sql-server-by-setting-a-storage-account -- 404
----------------------
Provider: Azure
Check: defender_ensure_defender_for_app_services_is_on.metadata.json
https://docs.prowler.com/checks/ensure-that-azure-defender-is-set-to-on-for-app-service#terraform -- 404
----------------------
Provider: Azure
Check: defender_additional_email_configured_with_a_security_contact.metadata.json
https://docs.prowler.com/checks/ensure-that-security-contact-emails-is-set#terraform -- 404
----------------------
Provider: Azure
Check: defender_ensure_defender_for_sql_servers_is_on.metadata.json
https://docs.prowler.com/checks/ensure-that-azure-defender-is-set-to-on-for-sql-servers-on-machines#terraform -- 404
----------------------
Provider: Azure
Check: defender_ensure_defender_for_containers_is_on.metadata.json
https://docs.prowler.com/checks/ensure-that-azure-defender-is-set-to-on-for-container-registries#terraform -- 404
----------------------
Provider: Azure
Check: defender_ensure_defender_for_azure_sql_databases_is_on.metadata.json
https://docs.prowler.com/checks/ensure-that-azure-defender-is-set-to-on-for-azure-sql-database-servers#terraform -- 404
----------------------
Provider: Azure
Check: defender_ensure_defender_for_storage_is_on.metadata.json
https://docs.prowler.com/checks/ensure-that-azure-defender-is-set-to-on-for-storage#terraform -- 404
----------------------
Provider: Azure
Check: defender_ensure_defender_for_keyvault_is_on.metadata.json
https://docs.prowler.com/checks/ensure-that-azure-defender-is-set-to-on-for-key-vault#terraform -- 404
----------------------
Provider: Azure
Check: defender_ensure_defender_for_server_is_on.metadata.json
https://docs.prowler.com/checks/ensure-that-azure-defender-is-set-to-on-for-servers#terraform -- 404
----------------------
Provider: Gcp
Check: kms_key_not_publicly_accessible.metadata.json
https://docs.prowler.com/checks/ensure-that-cloud-kms-cryptokeys-are-not-anonymously-or-publicly-accessible#terraform -- 404
----------------------
Provider: Gcp
Check: cloudstorage_bucket_log_retention_policy_lock.metadata.json
https://docs.prowler.com/checks/ensure-that-retention-policies-on-log-buckets-are-configured-using-bucket-lock#terraform -- 404
----------------------
Provider: Gcp
Check: bigquery_dataset_cmk_encryption.metadata.json
https://docs.prowler.com/checks/ensure-gcp-big-query-tables-are-encrypted-with-customer-supplied-encryption-keys-csek-1#terraform -- 404
----------------------
Provider: Gcp
Check: bigquery_table_cmk_encryption.metadata.json
https://docs.prowler.com/checks/ensure-gcp-big-query-tables-are-encrypted-with-customer-supplied-encryption-keys-csek#terraform -- 404
----------------------
Provider: Gcp
Check: iam_audit_logs_enabled.metadata.json
https://docs.prowler.com/checks/ensure-that-cloud-audit-logging-is-configured-properly-across-all-services-and-all-users-from-a-project#terraform -- 404
----------------------
Provider: Gcp
Check: gke_cluster_no_default_service_account.metadata.json
https://docs.prowler.com/checks/ensure-gke-clusters-are-not-running-using-the-compute-engine-default-service-account#terraform -- 404
----------------------
Provider: Gcp
Check: compute_network_not_legacy.metadata.json
https://docs.prowler.com/checks/ensure-legacy-networks-do-not-exist-for-a-project#terraform -- 404
----------------------
Provider: Gcp
Check: gcr_container_scanning_enabled.metadata.json
https://docs.prowler.com/checks/ensure-gcp-gcr-container-vulnerability-scanning-is-enabled#terraform -- 404
----------------------
Provider: Gcp
Check: dataproc_encrypted_with_cmks_disabled.metadata.json
https://docs.prowler.com/checks/ensure-gcp-dataproc-cluster-is-encrypted-with-customer-supplied-encryption-keys-cseks#terraform -- 404
----------------------
Provider: Aws
Check: kms_cmk_rotation_enabled.metadata.json
https://docs.prowler.com/checks/ensure-kms-have-rotation-policy#terraform -- 404
----------------------
Provider: Aws
Check: s3_bucket_kms_encryption.metadata.json
https://docs.prowler.com/checks/ensure-that-s3-buckets-are-encrypted-with-kms-by-default#terraform -- 404
----------------------
Provider: Aws
Check: s3_bucket_no_mfa_delete.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_s3_24#terraform -- 404
----------------------
Provider: Aws
Check: s3_bucket_public_access.metadata.json
https://docs.prowler.com/checks/s3-bucket-should-have-public-access-blocks-defaults-to-false-if-the-public-access-block-is-not-attached#terraform -- 404
----------------------
Provider: Aws
Check: s3_account_level_public_access_blocks.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_s3_21#cloudformation -- 404
----------------------
Provider: Aws
Check: s3_account_level_public_access_blocks.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_s3_21#terraform -- 404
----------------------
Provider: Aws
Check: s3_bucket_level_public_access_block.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_s3_20#terraform -- 404
----------------------
Provider: Aws
Check: s3_bucket_object_lock.metadata.json
https://docs.prowler.com/checks/ensure-that-s3-bucket-has-lock-configuration-enabled-by-default#cli-command -- 404
----------------------
Provider: Aws
Check: s3_bucket_object_lock.metadata.json
https://docs.prowler.com/checks/ensure-that-s3-bucket-has-lock-configuration-enabled-by-default#terraform -- 404
----------------------
Provider: Aws
Check: emr_cluster_publicly_accesible.metadata.json
https://docs.prowler.com/checks/ensure-that-amazon-emr-clusters-security-groups-are-not-open-to-the-world#terraform -- 404
----------------------
Provider: Aws
Check: sns_topics_not_publicly_accessible.metadata.json
https://docs.prowler.com/checks/ensure-sns-topic-policy-is-not-public-by-only-allowing-specific-services-or-principals-to-access-it#terraform -- 404
----------------------
Provider: Aws
Check: glacier_vaults_policy_public_access.metadata.json
https://docs.prowler.com/checks/ensure-glacier-vault-access-policy-is-not-public-by-only-allowing-specific-services-or-principals-to-access-it#terraform -- 404
----------------------
Provider: Aws
Check: dynamodb_tables_kms_cmk_encryption_enabled.metadata.json
https://docs.prowler.com/checks/ensure-that-dynamodb-tables-are-encrypted#terraform -- 404
----------------------
Provider: Aws
Check: workspaces_volume_encryption_enabled.metadata.json
https://docs.prowler.com/checks/ensure-that-workspace-root-volumes-are-encrypted#cloudformation -- 404
----------------------
Provider: Aws
Check: workspaces_volume_encryption_enabled.metadata.json
https://docs.prowler.com/checks/ensure-that-workspace-root-volumes-are-encrypted#terraform -- 404
----------------------
Provider: Aws
Check: sqs_queues_not_publicly_accessible.metadata.json
https://docs.prowler.com/checks/ensure-sqs-queue-policy-is-not-public-by-only-allowing-specific-services-or-principals-to-access-it#terraform -- 404
----------------------
Provider: Aws
Check: redshift_cluster_automatic_upgrades.metadata.json
https://docs.prowler.com/checks/ensure-that-redshift-clusters-allow-version-upgrade-by-default#terraform -- 404
----------------------
Provider: Aws
Check: awslambda_function_no_secrets_in_variables.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_secrets_3#cli-command -- 404
----------------------
Provider: Aws
Check: awslambda_function_no_secrets_in_variables.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_secrets_3#cloudformation -- 404
----------------------
Provider: Aws
Check: awslambda_function_no_secrets_in_variables.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_secrets_3#terraform -- 404
----------------------
Provider: Aws
Check: eks_cluster_network_policy_enabled.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_kubernetes_1#terraform -- 404
----------------------
Provider: Aws
Check: eks_cluster_kms_cmk_encryption_in_secrets_enabled.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_kubernetes_3#fix---builtime -- 404
----------------------
Provider: Aws
Check: eks_control_plane_logging_all_types_enabled.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_kubernetes_4#aws-console -- 404
----------------------
Provider: Aws
Check: eks_control_plane_logging_all_types_enabled.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_kubernetes_4#fix---buildtime -- 404
----------------------
Provider: Aws
Check: vpc_subnet_no_public_ip_by_default.metadata.json
https://docs.prowler.com/checks/ensure-vpc-subnets-do-not-assign-public-ip-by-default#terraform -- 404
----------------------
Provider: Aws
Check: guardduty_is_enabled.metadata.json
https://docs.prowler.com/checks/ensure-guardduty-is-enabled-to-specific-orgregion#fix---buildtime -- 404
----------------------
Provider: Aws
Check: cloudformation_stack_outputs_find_secrets.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_secrets_2#cli-command -- 404
----------------------
Provider: Aws
Check: rds_instance_enhanced_monitoring_enabled.metadata.json
https://docs.prowler.com/checks/ensure-that-enhanced-monitoring-is-enabled-for-amazon-rds-instances#terraform -- 404
----------------------
Provider: Aws
Check: rds_instance_deletion_protection.metadata.json
https://docs.prowler.com/checks/ensure-that-rds-clusters-and-instances-have-deletion-protection-enabled#terraform -- 404
----------------------
Provider: Aws
Check: rds_instance_backup_enabled.metadata.json
https://docs.prowler.com/checks/ensure-that-rds-instances-have-backup-policy#terraform -- 404
----------------------
Provider: Aws
Check: rds_instance_minor_version_upgrade_enabled.metadata.json
https://docs.prowler.com/checks/ensure-aws-db-instance-gets-all-minor-upgrades-automatically#cloudformation -- 404
----------------------
Provider: Aws
Check: rds_instance_minor_version_upgrade_enabled.metadata.json
https://docs.prowler.com/checks/ensure-aws-db-instance-gets-all-minor-upgrades-automatically#terraform -- 404
----------------------
Provider: Aws
Check: rds_instance_integration_cloudwatch_logs.metadata.json
https://docs.prowler.com/checks/ensure-that-respective-logs-of-amazon-relational-database-service-amazon-rds-are-enabled#terraform -- 404
----------------------
Provider: Aws
Check: apigateway_restapi_logging_enabled.metadata.json
https://docs.prowler.com/checks/ensure-api-gateway-stage-have-logging-level-defined-as-appropiate#terraform -- 404
----------------------
Provider: Aws
Check: sagemaker_notebook_instance_without_direct_internet_access_configured.metadata.json
https://docs.prowler.com/checks/ensure-that-direct-internet-access-is-disabled-for-an-amazon-sagemaker-notebook-instance#fix---buildtime -- 404
----------------------
Provider: Aws
Check: athena_workgroup_encryption.metadata.json
https://docs.prowler.com/checks/ensure-that-athena-workgroup-is-encrypted#terraform -- 404
----------------------
Provider: Aws
Check: ec2_instance_detailed_monitoring_enabled.metadata.json
https://docs.prowler.com/checks/ensure-that-detailed-monitoring-is-enabled-for-ec2-instances#terraform -- 404
----------------------
Provider: Aws
Check: ec2_instance_secrets_user_data.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_secrets_1#cli-command -- 404
----------------------
Provider: Aws
Check: ec2_instance_secrets_user_data.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_secrets_1#cloudformation -- 404
----------------------
Provider: Aws
Check: ec2_instance_secrets_user_data.metadata.json
https://docs.prowler.com/checks/aws/general-policies/bc_aws_secrets_1#terraform -- 404
----------------------
Provider: Aws
Check: ec2_ebs_default_encryption.metadata.json
https://docs.prowler.com/checks/ensure-ebs-default-encryption-is-enabled#aws-console -- 404
----------------------
Provider: Aws
Check: ec2_ebs_default_encryption.metadata.json
https://docs.prowler.com/checks/ensure-ebs-default-encryption-is-enabled#terraform -- 404
----------------------
Provider: Aws
Check: ec2_networkacl_allow_ingress_tcp_port_3389.metadata.json
https://docs.prowler.com/checks/ensure-aws-nacl-does-not-allow-ingress-from-00000-to-port-3389#cloudformation -- 404
----------------------
Provider: Aws
Check: ec2_networkacl_allow_ingress_tcp_port_3389.metadata.json
https://docs.prowler.com/checks/ensure-aws-nacl-does-not-allow-ingress-from-00000-to-port-3389#terraform -- 404
----------------------
Provider: Aws
Check: ec2_ebs_snapshots_encrypted.metadata.json
https://docs.prowler.com/checks/aws/general-policies/general_3-encrypt-eps-volume#cloudformation -- 404
----------------------
Provider: Aws
Check: ec2_ebs_snapshots_encrypted.metadata.json
https://docs.prowler.com/checks/aws/general-policies/general_3-encrypt-eps-volume#aws-console -- 404
----------------------
Provider: Aws
Check: ec2_ebs_snapshots_encrypted.metadata.json
https://docs.prowler.com/checks/aws/general-policies/general_3-encrypt-eps-volume#terraform -- 404
----------------------
Provider: Aws
Check: ec2_networkacl_allow_ingress_tcp_port_22.metadata.json
https://docs.prowler.com/checks/ensure-aws-nacl-does-not-allow-ingress-from-00000-to-port-22#cloudformation -- 404
----------------------
Provider: Aws
Check: ec2_networkacl_allow_ingress_tcp_port_22.metadata.json
https://docs.prowler.com/checks/ensure-aws-nacl-does-not-allow-ingress-from-00000-to-port-22#terraform -- 404
----------------------
Provider: Kubernetes
Check: rbac_minimize_secret_access.metadata.json
https://docs.prowler.com/checks/kubernetes/kubernetes-policy-index/kubernetes/kubernetes-policy-index/no-serviceaccountnode-should-be-able-to-read-all-secrets -- 404
----------------------
Provider: Kubernetes
Check: core_minimize_hostPID_containers.metadata.json
https://docs.prowler.com/checks/kubernetes/kubernetes-policy-index/kubernetes/kubernetes-policy-index/bc_k8s_1#kubernetes -- 404
----------------------
Provider: Kubernetes
Check: core_minimize_hostIPC_containers.metadata.json
https://docs.prowler.com/checks/kubernetes/kubernetes-policy-index/kubernetes/kubernetes-policy-index/bc_k8s_3#kubernetes -- 404
----------------------
Provider: Kubernetes
Check: core_minimize_root_containers_admission.metadata.json
https://docs.prowler.com/checks/kubernetes/kubernetes-policy-index/kubernetes/kubernetes-policy-index/bc_k8s_5#kubernetes -- 404
----------------------
Provider: Kubernetes
Check: core_no_secrets_envs.metadata.json
https://docs.prowler.com/checks/kubernetes/kubernetes-policy-index/kubernetes/kubernetes-policy-index/bc_k8s_33#kubernetes -- 404
----------------------
Provider: Kubernetes
Check: core_seccomp_profile_docker_default.metadata.json
https://docs.prowler.com/checks/kubernetes/kubernetes-policy-index/kubernetes/kubernetes-policy-index/bc_k8s_30#kubernetes -- 404
----------------------
Provider: Kubernetes
Check: core_minimize_hostNetwork_containers.metadata.json
https://docs.prowler.com/checks/kubernetes/kubernetes-policy-index/kubernetes/kubernetes-policy-index/bc_k8s_4#kubernetes -- 404
----------------------
Provider: Kubernetes
Check: core_minimize_privileged_containers.metadata.json
https://docs.prowler.com/checks/kubernetes/kubernetes-policy-index/kubernetes/kubernetes-policy-index/bc_k8s_2#kubernetes -- 404
----------------------
Provider: Kubernetes
Check: core_minimize_net_raw_capability_admission.metadata.json
https://docs.prowler.com/checks/kubernetes/kubernetes-policy-index/kubernetes/kubernetes-policy-index/bc_k8s_6#kubernetes -- 404
----------------------
Provider: Kubernetes
Check: core_minimize_admission_windows_hostprocess_containers.metadata.json
https://docs.prowler.com/checks/kubernetes/kubernetes-policy-index/kubernetes/kubernetes-policy-index/bc_k8s_1#kubernetes -- 404
----------------------
Provider: Kubernetes
Check: core_minimize_containers_capabilities_assigned.metadata.json
https://docs.prowler.com/checks/kubernetes/kubernetes-policy-index/kubernetes/kubernetes-policy-index/bc_k8s_34#kubernetes -- 404
----------------------
Provider: Kubernetes
Check: core_minimize_admission_hostport_containers.metadata.json
https://docs.prowler.com/checks/kubernetes/kubernetes-policy-index/kubernetes/kubernetes-policy-index/bc_k8s_25#kubernetes -- 404
----------------------
Provider: Kubernetes
Check: core_minimize_allowPrivilegeEscalation_containers.metadata.json
https://docs.prowler.com/checks/kubernetes/kubernetes-policy-index/kubernetes/kubernetes-policy-index/bc_k8s_19#kubernetes -- 404
----------------------
```

**Comment by MrCloudSec** [comment]
> https://docs.prowler.com/checks/aws/general-policies/general_3-encrypt-eps-volume

All of them fixed!

**Review by jfagoagas — APPROVED** [review]

**Pull request merged**